code/modules/jobs/job_types/lp_jobs.dm

```diff
@@ -0,0 +1,152 @@
+/datum/job/lp/lieutenant
```

Comment:
1. won't you want to define the job's outfits to be the ones you made?
2. shouldn't these be subtypes of their counterparts (like engineering specialist being a subtype of station engineer?)

Reply:
1. Understanding what you mean now, I've actually attached the outfit to the job.
2. Not for the LP, I wanted to make wholly unique jobs and outfits for them so I made them subtypes of /lp/.

Reply:
(At time of posting I'm testing the whole adding the outfit to the job)

Reply:
This should be fixed now.